samples/orbit-2-posts/src/main/java/com/babylon/orbit2/sample/posts/data/posts/network/AvatarUrlGenerator.kt

```diff
@@ -17,5 +17,5 @@
 package com.babylon.orbit2.sample.posts.data.posts.network
 
 class AvatarUrlGenerator {
-    fun generateUrl(email: String) = "https://api.adorable.io/avatars/285/$email"
+    fun generateUrl(email: String) = "https://i.pravatar.cc/285?u=$email"
```

Comment:
Changed service used as it makes the sample app look nicer...

<img width="643" alt="image" src="https://user-images.githubusercontent.com/182422/89349520-362c2780-d6a6-11ea-8bf6-968c507855a2.png">